src/app/controllers/functions_controller.rb

```diff
@@ -30,6 +30,10 @@ def edit; end
 
   def create
     @function = @user.functions.new(function_params)
+    tags = params[:tags]
+    @function.tags = tags.split(',').map do |tag|
+      Tag.find_or_create_by(name: tag.strip)
+    end
```

Comment:
that doesn't seem clean enough, while looking for a better solution this one came first result 
https://stackoverflow.com/questions/8345273/comma-separated-array-with-a-text-field-in-rails
it looks cleaner 
and will have one tag per tag name